### DISCUSSION TITLE
Make it not manage other installed applications

### DISCUSSION THREAD

**Discussion by he3als** [microsoft/winget-cli#3139] [Ideas]
Is it possible for WinGet to only update and manage packages that are installed with WinGet?

**Comment by Trenly** [comment]
> Is it possible for WinGet to only update and manage packages that are installed with WinGet?

Currently there is not a way to do this

**Reply by elisimpson** [reply]
I definitely would not want this as the default behavior. As an option it would be okay. Winget managing all installed apps, whether it installed them or not, is one of its best features.

**Comment by he3als** [comment]
Moved to ideas
